node/utils/simulation.ts

```diff
@@ -68,18 +68,13 @@ export const orderFormItemToSeller = (
     logisticsInfo: any[]
   }
 ) => {
-  const unitMultiplier = orderFormItem.unitMultiplier ?? 1
   const [logisticsInfo] = orderFormItem.logisticsInfo
 
+  const priceTags = orderFormItem.priceTags?.reduce(
+    (discount, priceTag: PriceTag) => discount + priceTag.value, 0)
```

Comment:
If `orderFormItem.priceTags` is not found, this variable will become `undefined` (due to the `?` operator). This will be a problem in line 75, because you would be adding `price + undefined`, which results in NaN.

So, just to be safe:

```suggestion
  const priceTags = orderFormItem.priceTags?.reduce(
    (discount, priceTag: PriceTag) => discount + priceTag.value, 0) ?? 0
```